Element.js

```diff
@@ -288,7 +299,12 @@ Element.prototype = {
 		}
 
 		// If the input isn't a filename, just post the value directly
-		return this._post('value', getPostData(value)).then(noop);
+		return this._post('value', getPostData(value)).then(noop).catch(function(error){
+			if(error.detail.error === 'invalid argument') {
```

Comment:
This should also check whether the `valueParameterCalledCheck` flag was already set. If the flag has been set, there's no point in setting it again and retrying the POST.

Reply:
Fixed